tests/test.py

```diff
@@ -1317,6 +1319,31 @@ def get_vault_cli_args(self, mount_path=None, secret_path=None):
         return args
 
 
+class AWSKMSClient:
+    def __init__(self):
+        self.url = os.environ.get('AWS_KMS_ADDRESS', 'http://localhost:8080')
+        self.kms_client = boto3.client('kms', aws_access_key_id='', aws_secret_access_key='', region_name='eu-west-1', endpoint_url=self.url)
+        # override request signer to skip boto3 looking for credentials in ~/.aws_credentials
+        self.kms_client._request_signer.sign = (lambda *args, **kwargs: None)
+
+    def get_kms_url(self):
+        return self.url
+
+    def create_key(self):
+        response = self.kms_client.create_key(Description='AcraMasterKey KEK')
+        return response['KeyMetadata']['Arn']
+
+    def disable_key(self, keyId):
+        self.kms_client.disable_key(KeyId=keyId)
+
+    def generate_master_key(self, keyId):
+        response = self.kms_client.generate_data_key(
```

Comment:
hm, can we generate random master key and encrypt it via KMS as it actually will be used in the acra? I understand, that the logically generation data key from KMS side should be the same. But we met situations when 3d party products broke their API and much better if we can detect changes of API from which we depend. Plus, we may use real AWS API for tests in the future if meet situations when the emulator is not enough and it may charge extra resources for +1 kind of key and different api call